packages/@react-types/form/src/index.d.ts

```diff
@@ -14,11 +14,21 @@ import {DOMProps, SpectrumLabelableProps, StyleProps, ValidationState} from '@re
 import {ReactElement} from 'react';
 
 export interface SpectrumFormProps extends DOMProps, StyleProps, SpectrumLabelableProps {
+  /** The contents of the form. */
   children: ReactElement<SpectrumLabelableProps> | ReactElement<SpectrumLabelableProps>[],
+  /** Whether the form elements are displayed with their quiet style. */
   isQuiet?: boolean,
+  /** Whether the form elements are rendered with their emphasized style (blue). */
```

Comment:
I think I've said the color blue too, but is this an implementation detail that we shouldn't mention?

Reply:
Yeah, that sounds like a good idea. I'll remove elsewhere as well.